client/src/lib.rs

```diff
@@ -21,27 +21,27 @@
 
 // Rustc
 #![warn(trivial_casts)]
-// #![deny(
-// 	absolute_paths_not_starting_with_crate, deprecated, future_incompatible, missing_docs,
-// 	nonstandard_style, unreachable_code, unreachable_patterns
-// )]
+#![deny(
+	absolute_paths_not_starting_with_crate, deprecated, future_incompatible, nonstandard_style,
+	unreachable_code, unreachable_patterns
+)]
 #![forbid(unsafe_code)]
 // Clippy
 // #![allow(clippy::tabs_in_doc_comments)]
```

Comment:
We could remove this rule from the client, wdyt?

Reply:
Sure, I don't see the need for keeping this atm

Reply:
@4rmanc Can you remove this for now, so we can merge this?

Reply:
@brech1 Are you talking about clippy::tabs_in_doc_comments?

Reply:
Yes 👍🏼